dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphIntegrationTest.java

```diff
@@ -74,8 +74,12 @@ public void testFindUpdates()
         + "upgrade com.google.http-client:google-http-client:1.19.0 to 1.23.0");
   }
   
+  // Beam has a more complex dependency graph that hits some corner cases.
+  // In particular it pulls in Netty, which pulls in native code, the 
+  // exact artifact depending on which operating system you're running on.
+  // This is the static block that sets the os.detected.classifier system property.
```

Comment:
Which is the static block that "this" refers to? Take *what* out and this test will fail?

Can you add a comment, maybe on line 90, like `// should not throw`?

Reply:
done